setup.py

```diff
@@ -99,7 +120,15 @@ def get_suppressed_warning_flags():
         return []
 
 
+def build_backend():
+    if IS_LIN:
+        subprocess.check_call(["/bin/bash", "-c", "scripts/build_backend.sh"])
+    elif IS_WIN:
+        subprocess.check_call(["cmd.exe", "/c", "scripts\\build_backend.bat"])
```

Comment:
If build scripts are Python then no need to have 2 versions for Linux and Windows.